www/src/mapper/Thumbnail.tsx

```diff
@@ -93,23 +101,24 @@ export let Thumb = (props: {
             </div>
             }
           </div>
-          {loaded &&
-          <>
           {/* the generated image might not be exactly square, so use a sized container div and make the img `w-full h-full` */}
           <div
-            className="col-span-full row-span-full animate-quickfadein"
-            style={{height: height, width: width}}
+            className="col-span-full row-span-full"
+            style={{height: height, width: width, visibility: loaded? 'visible' : 'hidden'}}
           >
             <img
               src={src}
               className="w-full h-full rounded-md"
               alt={`Thumbnail image for ${getDisplayDate(props.date)}`}
+              onLoad={(e) => {
+                let imageElement = e.target as HTMLElement
+                imageElement.classList.add('animate-quickfadein') // trigger animation when visible
```

Comment:
We shouldn't break out of React if we can help it - shall we have a quick look together at this after merge